components/04-views/catalog/_catalog.scss

```diff
@@ -1,37 +1,55 @@
+//product-grid
+$product-grid-padding             : 10px !default;
+$product-grid-margin              : 15px !default;
+$product-grid-breakpoint-medium   : $screen-m !default;
+$product-grid-breakpoint-large    : $screen-l !default;
+
+//product-list
+$product-list-padding             : 10px !default;
+$product-list-margin              : 10px !default;
+$product-list-border-color        : $gray-light !default;
+$product-list-border              : 1px solid $product-list-border-color !default;
+$product-list-name-font-size      : $font-size-small !default;
+$product-list-details-color       : $gray-lighter !default;
+$product-list-details-font-size   : $font-size-extra-small !default;
+$product-list-img-box-padding     : 0 $product-grid-padding 0 0 !default;
+$product-list-img-box-border-color: $gray-light !default;
+$product-list-img-box-border      : 1px solid $product-list-img-box-border-color !default;
+
 .products-grid {
     display: flex;
     flex-flow: row wrap;
+    justify-content: flex-start;
     width: 100%;
     margin: 0;
     padding: 0;
-    justify-content: flex-start;
     list-style-type: none;
 
-    @include mq($screen-m) {
-        padding: 10px 0 10px 20px;
+    @include mq($product-grid-breakpoint-medium) {
+        padding: $product-grid-padding 0 $product-grid-padding ($product-grid-padding * 2);
     }
 
     &--4-in-row {
         .products-grid__item {
-            @include mq($screen-l) {
+            @include mq($product-grid-breakpoint-large) {
                 flex-basis: calc(25% - 20px);
             }
         }
     }
 
     &__item {
         flex-basis: 100%;
-        margin-bottom: 15px;
+        margin-bottom: $product-grid-margin;
 
         @include mq($screen-s) {
             flex-basis: calc(50% - 15px);
-            margin-right: 15px;
+            margin-right: $product-grid-margin;
         }
 
-        @include mq($screen-m) {
+        @include mq($product-grid-breakpoint-medium) {
             flex-basis: calc(33.33% - 20px);
-            margin-bottom: 20px;
-            margin-right: 20px;
+            margin-bottom: ($product-grid-margin + 5);
```

Comment:
General comment about all variables that are used in many places but with some math around it - I'm not sure about it, b/c it might be kind frustrating that the "core" value affect child elements in some, kinda random, way.

Reply:
Generally, I checked all variables with some math and in most cases, I've deleted it. Left only this which i think it's related